### DISCUSSION TITLE
Preprint or Paper on actual Modeling

### DISCUSSION THREAD

**Discussion by tleonard6** [neherlab/covid19_scenarios#598] [General]
Hello, is there a paper detailing the actual solution method of the system of equations presented in the about section?  I'm trying to replicate the solution technique for a more granular area county level versus state level for the U.S..

https://covid19-scenarios.org/about

**Comment by rneher** [comment]
we are working on a preprint. but we aren't quite ready yet....

**Comment by lewleib** [comment]
I started to read Neher et. al " Potential impact of seasonal forcing on a SARS-CoV-2 pandemic".  However, I wasn't able to download the Appendix.  The browser kept going in circles.
Any suggestions?

**Reply by rneher** [reply]
works fine for me: 
[SMW_20224_Appendix.pdf](https://github.com/neherlab/covid19_scenarios/files/4548255/SMW_20224_Appendix.pdf)

**Comment by tleonard6** [comment]
@rneher, @lewleib Thank you, that is helpful

**Comment by lewleib** [comment]
Thank you! That worked.

Regards
@lewleib
